Mlem/App/Views/Root/Tabs/Feeds/SubscriptionListView.swift

```diff
@@ -28,6 +29,13 @@ struct SubscriptionListView: View {
         })
         .navigationTitle("Feeds")
         .navigationBarTitleDisplayMode(.inline)
+        .onChange(of: appState.firstApi, initial: true) {
+            if appState.firstApi.canInteract, appState.firstAccount is UserAccount {
```

Comment:
`appState.firstAccount is UserAccount` is sufficient here
```suggestion
            if appState.firstAccount is UserAccount {
```